x-pack/plugins/observability_solution/slo/public/pages/slos/components/slos_overview/slos_overview.tsx

```diff
@@ -0,0 +1,123 @@
+  const { state } = useUrlSearchState();
+  const { kqlQuery, filters, tagsFilter, statusFilter } = state;
+
+  const { data, isLoading } = useFetchSLOsOverview({
+    kqlQuery,
+    filters,
+    tagsFilter,
+    statusFilter,
+  });
+
+  const theme = useEuiTheme().euiTheme;
+  const { data: currentSettings } = useGetSettings();
+
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem grow={2}>
+        <EuiPanel hasShadow={false} hasBorder={true}>
+          <EuiTitle size="xs">
+            <h3>
+              {i18n.translate('xpack.slo.sLOsOverview.h3.overviewLabel', {
+                defaultMessage: 'Overview',
+              })}
+            </h3>
+          </EuiTitle>
+          <EuiSpacer size="xs" />
+          <EuiFlexGroup gutterSize="xl" justifyContent="spaceBetween">
+            <OverViewItem
+              title={data?.healthy}
+              description={i18n.translate('xpack.slo.sLOsOverview.euiStat.healthyLabel', {
+                defaultMessage: 'Healthy',
+              })}
+              titleColor="success"
+              isLoading={isLoading}
+              query={`status : HEALTHY`}
+              tooltip={i18n.translate('xpack.slo.sLOsOverview.euiStat.healthyLabel.tooltip', {
+                defaultMessage: 'Click to filter SLOs by Healthy status.',
+              })}
+            />
+            <OverViewItem
+              title={data?.violated}
+              description={i18n.translate('xpack.slo.sLOsOverview.euiStat.violatedLabel', {
+                defaultMessage: 'Violated',
+              })}
+              titleColor="danger"
+              query={`status : VIOLATED`}
+              isLoading={isLoading}
+              tooltip={i18n.translate('xpack.slo.sLOsOverview.euiStat.violatedLabel.tooltip', {
+                defaultMessage: 'Click to filter SLOs by Violated status.',
+              })}
+            />
+            <OverViewItem
+              title={data?.noData}
+              description={i18n.translate('xpack.slo.sLOsOverview.euiStat.noDataLabel', {
+                defaultMessage: 'No data',
+              })}
+              titleColor="subdued"
+              query={`status : NO_DATA`}
+              isLoading={isLoading}
+              tooltip={i18n.translate('xpack.slo.sLOsOverview.euiStat.noDataLabel.tooltip', {
+                defaultMessage: 'Click to filter SLOs by no data status.',
+              })}
+            />
+            <OverViewItem
+              title={data?.degrading}
+              description={i18n.translate('xpack.slo.sLOsOverview.euiStat.degradingLabel', {
+                defaultMessage: 'Degrading',
+              })}
+              query={`status : DEGRADING`}
+              isLoading={isLoading}
+              tooltip={i18n.translate('xpack.slo.sLOsOverview.euiStat.degradingLabel.tooltip', {
+                defaultMessage: 'Click to filter SLOs by Degrading status.',
+              })}
+              titleColor={theme.colors.warningText}
+            />
```

Comment:
Are we removing the filter from the search bar in order to keep only this way of filtering?
If not, what happens if a user selects Healthy from the main search filter bar, and degrading here? Does one take precedence on the other? or both are applied?

Reply:
both filters are applied, this just updates the kql filter on click